tests/unit/conftest.py

```diff
@@ -0,0 +1,23 @@
+# This file is part of craft_application.
+#
+# Copyright 2023 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 3, as published
```

Comment:
This should be LGPL instead of GPL.

Reply:
Good catch, thanks!

It looks like I've got some other spots that are incorrect too so I'll make a separate PR to get the others changed in bulk.